dockerfiles/ci/Dockerfile

```diff
@@ -12,17 +12,16 @@
 # Dockerfile defines che-machine-exec production image eclipse/che-machine-exec-dev
 #
 
-FROM golang:1.10.3-alpine as builder
+FROM golang:1.10.7-alpine as builder
 WORKDIR /go/src/github.com/eclipse/che-machine-exec/
 COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -a -ldflags '-w -s' -a -installsuffix cgo -o che-machine-exec .
 RUN apk add --no-cache ca-certificates
 
 RUN adduser -D -g '' unprivilegeduser
```

Comment:
what's the meaning of this instruction as we're in another image/container at runtime ?

Reply:
Hello this change was not mandatory. I reverted it.